pkg/config/node.go

```diff
@@ -167,9 +171,13 @@ func GetConfig(kubeconfigPath, clusterConfigPath, resolvConfPath string, apiVip
 		node.Cluster.DNSVirtualRouterID++
 	}
 	node.Cluster.IngressVirtualRouterID = utils.FletcherChecksum8(node.Cluster.Name+"-ingress") + 1
-	for node.Cluster.IngressVirtualRouterID == node.Cluster.DNSVirtualRouterID || node.Cluster.IngressVirtualRouterID == node.Cluster.APIVirtualRouterID {
+	if node.Cluster.IngressVirtualRouterID == node.Cluster.DNSVirtualRouterID || node.Cluster.IngressVirtualRouterID == node.Cluster.APIVirtualRouterID {
 		node.Cluster.IngressVirtualRouterID++
 	}
+	node.Cluster.APIProvisioningVirtualRouterID = utils.FletcherChecksum8(node.Cluster.Name+"-api-provisioning") + 1
+	if node.Cluster.APIProvisioningVirtualRouterID == node.Cluster.IngressVirtualRouterID || node.Cluster.APIProvisioningVirtualRouterID == node.Cluster.DNSVirtualRouterID || node.Cluster.APIProvisioningVirtualRouterID == node.Cluster.APIVirtualRouterID {
```

Comment:
I don't think that it's mandatory to verify that PROV_VIP overlaps the rest of VIPs because it will run on another network interface.

Reply:
The Router ID still needs to be unique, doesn't it?

Reply:
Ooop, I forgot to add Router ID to my previous comment. I think that Router IDs should be unique only for VRRP sessions that run on the same network interface, so since this VRRP session will run on a different network interface (the provisioning net interface) it isn't mandatory to verify that PROV router ID is unique

Reply:
That's fair.